code/modules/mob/transform_procs.dm

```diff
@@ -461,7 +461,10 @@
 	qdel(src)
 
 /mob/proc/become_overmind(starting_points = 60)
-	var/mob/camera/blob/B = new /mob/camera/blob(loc, starting_points)
+	var/turf/T = get_turf(loc) //just to avoid messing up in lockers

```

Comment:
Just check the area to see if its blob safe to stop them spawning in space or the ops shuttle or wherever as well

Reply:
I made it _also_ check to see if it's blob safe